code/modules/detectivework/tools/swabs.dm

```diff
@@ -126,3 +139,4 @@
 	desc = "[initial(desc)] The label on the vial reads 'Sample of [sample_str] from [source].'."
 	icon_state = "swab_used"
 	used = 1
+	collecting_evidence = 0
```

Comment:
I kinda want to pester you to change the name. "Collecting evidence" implies that the variable is used to determine whether or not the user is actively collecting evidence. (In the process of doing so at this very moment.) Where as the actual semantics are more in line with "accepting evidence".